route/route.go

```diff
@@ -203,114 +202,24 @@ func (r *Router) version(w http.ResponseWriter, req *http.Request) {
 func (r *Router) event(w http.ResponseWriter, req *http.Request) {
 	defer req.Body.Close()
 
-	// get out request ID for logging
-	reqID := req.Context().Value(types.RequestIDContextKey{})
-	debugLog := r.iopLogger.Debug().WithField("request_id", reqID)
```

Comment:
we removed this from here but not from the batch endpoint, should it be only in the batch endpoint?

Reply:
good catch, I put it back